### PR TITLE
CLOUDP-66831: mongocli atlas cluster(s) onlineArchive list

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ [CLOUDP-66831](https://jira.mongodb.org/browse/CLOUDP-66831)

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Example
```bash
mongocli atlas clusters onlinearchive ls --clusterName testShard
[
	{
		"_id": "5f04b0e13fe0b32bb2d78af8",
		"clusterName": "testShard",
		"collName": "test",
		"criteria": {
			"dateField": "created_at",
			"expireAfterDays": 2
		},
		"dbName": "test",
		"groupId": "5e429f2e06822c6eac4d59c9",
		"paused": false,
		"state": "PENDING"
	}
]
```

### CODE REVIEW

File: internal/cli/atlas/clusters/clusters.go [hunk 1]
```diff
@@ -12,30 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package atlas
+package clusters
```
Comment (gssbzn) [inline internal/cli/atlas/clusters/clusters.go:15]: moved cluster to its own package, may do a separate PR if it makes easier to review this one

**Comment by gssbzn** [comment]
Depends on https://github.com/mongodb/go-client-mongodb-atlas/pull/107

**Comment by gssbzn** [comment]
This should be ready to review @robcarlan-mlab

File: internal/cli/atlas/clusters/clusters.go (hunk 1 above)
Comment (robcarlan-mlab) [reply internal/cli/atlas/clusters/clusters.go:15]: 👍

File: internal/cli/atlas/onlinearchive/list.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+		Args:    cobra.NoArgs,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.initStore()
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringVar(&opts.clusterName, flag.ClusterName, "", usage.Page)
```
Comment (robcarlan-mlab) [inline internal/cli/atlas/onlinearchive/list.go:65]: `usage.Page` is a copy paste error?

Comment (gssbzn) [reply internal/cli/atlas/onlinearchive/list.go]: yep, fixed
